marlowe-cli/cli-test/Language/Marlowe/CLI/Test/Runner.hs

```diff
@@ -204,7 +204,7 @@ data Env lang era resource = Env
   , _envStreamJSON :: Bool
   , _envSlotConfig :: SlotConfig
   , _envProtocolParams :: CS.ProtocolParameters
-  , _envCostModelParams :: P.CostModelParams
+  , _envCostModelParams :: [Integer]
```

Comment:
So fragile, but we're stuck with this for now. My understanding is that it will change with a future release.

Reply:
Yeah, this is super confusing